polygon/bridge/bridge.go

```diff
@@ -63,6 +63,7 @@ func (b *Bridge) Run(ctx context.Context) error {
 	for {
 		select {
 		case <-ctx.Done():
+			b.Close()
```

Comment:
can use `defer b.Close()` at the beginning of this func and remove all the `b.Close()` calls here and below